intelij-plugin/src/main/java/tech/pantheon/yanginator/plugin/formatter/settings/YangLanguageCodeStyleSettingsProvider.java

```diff
@@ -58,14 +57,14 @@ public void customizeSettings(@NotNull final CodeStyleSettingsCustomizable consu
 
     @Override
     public IndentOptionsEditor getIndentOptionsEditor() {
-        return new SmartIndentOptionsEditor();
+        return new IndentOptionsEditor();
     }
 
     @Override
     protected void customizeDefaults(@NotNull final CommonCodeStyleSettings commonSettings,
                                      @NotNull final CommonCodeStyleSettings.IndentOptions indentOptions) {
-        indentOptions.INDENT_SIZE = commonSettings.initIndentOptions().INDENT_SIZE;
-        indentOptions.CONTINUATION_INDENT_SIZE = commonSettings.initIndentOptions().CONTINUATION_INDENT_SIZE;
-        indentOptions.TAB_SIZE = commonSettings.initIndentOptions().TAB_SIZE;
+        indentOptions.INDENT_SIZE = 4;
```

Comment:
this is hardcoded now? It will not be possible to use custom indents.

Reply:
It is still possible to use custom indents. This just initialize the indents in the first place and if the user chooses something else it uses indents user chose.

Reply:
I see it now. The continuation indent size value should be the same as the indent size value to maintain proper indentation. Should I just add this information to the preview for the indentation code style page?